### PR TITLE
Dịch "gan_vn" - Phần 4

### LINKED ISSUE #2730: Dịch "gan_vn" - Phần 4
Phần cần dịch: https://github.com/aivivn/d2l-vn/blame/master/chapter_generative-adversarial-networks/gan_vn.md#L390-L512

### Hướng dẫn
- [Dịch trực tiếp trên website](https://github.com/aivivn/d2l-vn/blob/master/CONTRIBUTING.md).
- Sau khi nhấn vào phần cần dịch ở trên, hãy chọn **Normal view** và làm theo hướng dẫn ở trên để có thể bắt đầu dịch.

### PR DESCRIPTION
**Pull request by rootonchair**
## Pull Request Checklist

**Đánh dấu tick (sau khi "Create Pull Request") vào các mục dưới đây:**

* [x] Pull Request này tương ứng với issue nào? Điền số issue sau dấu `#` (không có dấu cách): Close #2730 

* [x] Bản dịch này có bám sát **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)** không? Nếu một từ/cụm chưa có trong Bảng thuật ngữ, hãy thảo luận thêm ở thẻ Discussions của dự án.

* [x] Format của bản gốc có được giữ nguyên không?
     
* [x] Trong một đoạn văn, mỗi câu văn đã được viết trong một dòng, giữa các dòng không có dòng trắng. Markdown sẽ coi những dòng liền nhau không có dòng trắng là một đoạn văn. Việc này giúp công đoạn review được thuận tiện hơn. Bạn đã kiểm tra rằng mỗi đoạn văn đã nằm một dòng?

* [x] Điền tên đầy đủ của bạn vào mục **Những người thực hiện** ở cuối tập tin.

## Lưu ý cho người phản biện (Reviewer)

1. Chỉ review khi Pull Request (PR) này đã sẵn sàng cho việc review. Một PR sẵn sàng khi PR đó không có label `status: wip`.
2. Đã đọc kỹ hướng dẫn đóng góp và xem qua **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)**.

### CODE REVIEW

File: chapter_generative-adversarial-networks/gan_vn.md [hunk 1]
```diff
@@ -396,7 +396,10 @@ In each iteration, we first update the discriminator and then the generator.
 We visualize both losses and generated examples.
 -->
 
-*dịch đoạn phía trên*
+Cả bộ phân biệt lẫn bộ sinh biểu diễn như một bộ hồi quy logistic nhị phân với mất mát entropy chéo.
```
Comment (cuongvng) [inline chapter_generative-adversarial-networks/gan_vn.md:399]:
```suggestion
Cả bộ phân biệt lẫn bộ sinh hoạt động như một bộ hồi quy logistic nhị phân với mất mát entropy chéo.
```

**Review by cuongvng — APPROVED** [review]

**Review by duythanhvn — APPROVED** [review]

File: chapter_generative-adversarial-networks/gan_vn.md [hunk 2]
```diff
@@ -497,7 +500,9 @@ train(net_D, net_G, data_iter, num_epochs, lr_D, lr_G,
 * The discriminator tries to distinguish the generated images from the true images, via minimizing the cross-entropy loss, *i.e.*, $\min - y \log D(\mathbf{x}) - (1-y)\log(1-D(\mathbf{x}))$.
 -->
 
-*dịch đoạn phía trên*
+* Mạng đối sinh (*Generative adversarial networks - GANs*) được cấu thành bởi hai mạng sâu, bộ sinh và bộ phân biệt.
+* Bộ sinh tạo các ảnh gần với ảnh thật nhất có thể nhằm đánh lừa bộ phân biệt, thông qua tối đa hóa mất mát entropy chéo, *nói cách khác*, $\max \log(D(\mathbf{x'}))$.
```
Comment (minhduc0711) [inline chapter_generative-adversarial-networks/gan_vn.md:504]:
```suggestion
* Bộ sinh tạo các ảnh gần với ảnh thật nhất có thể nhằm đánh lừa bộ phân biệt, thông qua việc tối đa hóa mất mát entropy chéo, *nói cách khác*, $\max \log(D(\mathbf{x'}))$.
```

File: chapter_generative-adversarial-networks/gan_vn.md [hunk 3]
```diff
@@ -497,7 +500,9 @@ train(net_D, net_G, data_iter, num_epochs, lr_D, lr_G,
...
+* Bộ phân biệt cố gắng phân biệt những ảnh được tạo với ảnh thật, thông qua tối thiểu hóa mất mát entropy chéo, *nói cách khác*, $\min - y \log D(\mathbf{x}) - (1-y)\log(1-D(\mathbf{x}))$.
```
Comment (minhduc0711) [inline chapter_generative-adversarial-networks/gan_vn.md:505]:
```suggestion
* Bộ phân biệt cố gắng phân biệt những ảnh được tạo với ảnh thật, thông qua việc tối thiểu hóa mất mát entropy chéo, *nói cách khác*, $\min - y \log D(\mathbf{x}) - (1-y)\log(1-D(\mathbf{x}))$.
```

**Review by minhduc0711 — APPROVED** [review]
